app/frontend/src/service/user.ts

```diff
@@ -129,6 +130,7 @@ export async function updateProfile(
 ): Promise<Empty> {
   const req = new UpdateProfileReq();
 
+  const avatarKey = new wrappers.StringValue().setValue(profile.avatarKey);
```

Comment:
```suggestion
  const avatarKey = profile.avatarKey ? new wrappers.StringValue().setValue(profile.avatarKey) : undefined;
```

This might solve this - the purpose of the wrapper is to differentiate between unsetting (ie. setting to nothing - wrapper is present but value is null) and just not making any changes (wrapper itself not present). This should do since you can't clear the image at this point.

Reply:
I've changed it to your suggestion